canonical-websites/www.ubuntu.com
# Update the OpenStack storage page

## Done
Updated the OpenStack storage page.

## QA
- Check out this feature branch
- Run the site using the command `./run serve`
- View the site locally in your web browser at: [http://0.0.0.0:8001/openstack/storage](http://0.0.0.0:8001/openstack/storage)
- Check that the page matches the [copy doc](https://docs.google.com/document/d/1iw2SWNmoj4GZ2pUh0yB8gbtVXK0oXfyEaXT44bQbPlI/edit#)

## Issue / Card
Fixes https://github.com/ubuntudesign/web-squad/issues/641
